ci-changes.md

```diff
@@ -0,0 +1,20 @@
+The following would be added to the `CI & Automation` Wiki page as a new section: https://github.com/camunda/camunda/wiki/CI-&-Automation
+
+
+# CI Test Files
+
+## CI Ownership
+Each CI test file has an owning team. The owning team can be found either through the `CODEOWNERS` file or on the metadata in the file itself. The `CODEOWNERS` file is organized and broken down by team, any additions to the file should follow that convention. The metadata on a CI file is used by a scraping tool so that it is easy to gather information about the current state of CI. You can look at the metadata for a quick overview of the owning team, where the tests live, how the test is called, and a description of what the file is actually testing
```

Comment:
Terminology: I'd use "GHA workflow file" instead of "CI file" since that is [in line with GH docs](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions)

Reply:
> used by a scraping tool

Once available, it would be good to have a link to the tool here.

Reply:
Maybe give an example (code snippet) or link to one for how the metadata looks like